project/Dependencies.scala

```diff
@@ -107,14 +106,13 @@ object Dependencies {
       val commonsCodec = "commons-codec" % "commons-codec" % "1.14" % "test" // ApacheV2
       val junit = "junit" % "junit" % junitVersion % "test" // Common Public License 1.0
       val logback = Compile.logback % "test" // EPL 1.0
-      val mockito = "org.mockito" % "mockito-core" % "3.2.4" % "test" // MIT
       // changing the scalatest dependency must be reflected in akka-docs/rst/dev/multi-jvm-testing.rst
```

Comment:
I think we can remove this comment, we don't use .rst-files anymore and the corresponding doc file doesn't have anything about ScalaTest dependency